lib/Yield/Pipes/AssemblyPipes.cpp

```diff
@@ -75,9 +75,10 @@ void YieldAssembly::run(pipeline::ExecutionContext &Context,
   for (auto [Address, S] : Input) {
     auto MaybeFunction = TupleTree<yield::Function>::deserialize(S);
     revng_assert(MaybeFunction && MaybeFunction->verify());
-    revng_assert((*MaybeFunction)->Entry() == Address);
+    revng_assert((*MaybeFunction)->Entry() == std::get<0>(Address));
```

Comment:
Mmmh, why do we need `std::get<0>` here? Is this a single-element tuple?

Reply:
Instead of `MetaAddress` we how get `std::tuple<MetaAddress>` (because it's uniform w.r.t. model keys)

Reply:
Sad but OK